### DISCUSSION TITLE
ahooks/

### DISCUSSION THREAD

**Discussion by giscus** [OweQian/oweqian.github.io#15] [General]
# ahooks/

2023.11 ~ 2024.02，历时 4 个月，在公司宣布 996.ICU(:不是，996 是打工人的福报) 期间读了 ahooks@3.7.8 官网和源码，以下是整理的笔记，欢迎您的指正以及贡献。

https://oweqian.xyz/ahooks/

**Comment by EricZZZ** [comment]
太长不看😂

**Reply by OweQian** [reply]
快来一起卷 React

**Comment by GOWxx** [comment]
太强了倩总！

**Reply by OweQian** [reply]
2024，改变风气，争做卷逼

**Reply by GOWxx** [reply]
语录贡献 +1
